scripts/get-cdnjs-list.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import json
+import subprocess
+
+links = {}
+data = subprocess.check_output(['links', '-dump', 'https://cdnjs.com/'])
```

Comment:
@Aeyoun I don't understand. The list of available libraries cdnjs supports is quite extensive.